haaska.py

```diff
@@ -364,6 +402,15 @@ def set_percentage(self, val):
         brightness = (val / 100.0) * 255.0
         self._call_service('light/turn_on', {'brightness': brightness})
 
+    def set_color(self, hue, saturation, brightness):
+        red, green, blue = colorsys.hsv_to_rgb(hue, saturation, brightness)
```

Comment:
colorsys's documentation says

> In the YIQ space, the Y coordinate is between 0 and 1, but the I and Q coordinates can be positive or negative. In all other spaces, the coordinates are all between 0 and 1.

but Amazon's documentation for the hue field says:

> Valid range is 0.00 to 360.00, inclusive.

So I think some scaling needs to be done here, right?